pegin/pegin.go

```diff
@@ -42,7 +41,7 @@ func PeginAddress(
 	}
 
 	var mainChainAddress string
-	if isDynaFedEnabled || address.IsScriptHash(pops) {
+	if !isDynaFedEnabled || address.IsScriptHash(pops) { //TODO check IsScriptHash
```

Comment:
regression?

Reply:
@sekulicd

Reply:
need to double check address.IsScriptHash against elements implementation.
!isDynaFedEnabled is ok.

Reply:
Verifying IsScriptHash Done, this line should be ok.